### DISCUSSION TITLE
[讨论] 如何提高学习和交流的效率

### DISCUSSION THREAD

**Discussion by luoxiao78** [lczzlczz/TDD-in-Action#6] [General]
我先抛砖引玉 
### 使用 slack 作为团队交流工具:
- 在 slack 中为各小组创建单独的频道
- 每周在项目频道定期组织公共会议, 共同讨论, 周期可以由大家投票决定
- 各小组在公共会议前将小组近期的学习心得或问题在项目频道分享, 以便讨论
- 单独创建一个频道用于提问, 鼓励大家互相帮助, 并通过点赞推举自己认可的答案
- 回答问题时请创建或使用该问题的 Thread, 而不是在频道中直接回复
- 尽量减少无意义的发言, 并将发言内容压缩在一条消息内, 避免连续多条短信息
- 使用 slack 自带的 emoji reaction 来替代简短的回复, 比如'赞同', '强', '+1' 等

**Comment by lczzlczz** [comment]
> 我先抛砖引玉
> 
> ### 使用 slack 作为团队交流工具:
> * 在 slack 中为各小组创建单独的频道
> * 每周在项目频道定期组织公共会议, 共同讨论, 周期可以由大家投票决定
> * 各小组在公共会议前将小组近期的学习心得或问题在项目频道分享, 以便讨论
> * 单独创建一个频道用于提问, 鼓励大家互相帮助, 并通过点赞推举自己认可的答案
> * 回答问题时请创建或使用该问题的 Thread, 而不是在频道中直接回复
> * 尽量减少无意义的发言, 并将发言内容压缩在一条消息内, 避免连续多条短信息
> * 使用 slack 自带的 emoji reaction 来替代简短的回复, 比如'赞同', '强', '+1' 等

- 其实slack中是可以给用户分组的，用户分组之后效率很高，但是用户分组是付费版的内容，我们可以采用`人肉分组`的方式，我会在学习计划里提到。
- 就算分组了，我也不建议私下讨论问题，这违反了slack的原则。slack的快速检索能力可以让我们在遇到重复问题时，可以快速定位问题。
- 至少会有这些定期讨论:
  - `Weekly standup`: 每周一次，每个人说说上周完成了什么内容，并给出相应链接
  - `Tech round table`: 组员可以把学习的心得分享到这里，每周或每两周一次。 
- 其他都赞同。
